common/djangoapps/philu_commands/management/commands/tests/test_auto_generate_certificates_for_open_courses.py

```diff
@@ -0,0 +1,133 @@
+"""
+Tests for auto generate certificate for open courses
+"""
+from __future__ import unicode_literals
+
+from datetime import datetime, timedelta
+
+import pytz
+from django.core.management import call_command
+from django.db.models import signals
+from factory.django import mute_signals
+from mock import Mock, patch
+
+from lms.djangoapps.onboarding.tests.factories import UserFactory
+from philu_commands.management.commands.auto_generate_certificates_for_open_courses import (
+    is_course_valid_for_certificate_auto_generation
+)
+from student.tests.factories import CourseEnrollmentFactory
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+from xmodule.modulestore.tests.factories import CourseFactory
+
+
+class TestAutoGenerateCertificateForOpenCourse(ModuleStoreTestCase):
+    """
+    Tests for `auto_generate_certificates_for_open_courses.py` command.
+    """
+
+    @mute_signals(signals.pre_save, signals.post_save)
+    def setUp(self):
+        """
+        This function is responsible for creating courses for every test and mocking the function for tests.
+        """
+        super(TestAutoGenerateCertificateForOpenCourse, self).setUp()
+        self.course_1 = CourseFactory.create(display_name='test course 1', run='Testing_course_1')
+        self.course_1.end = datetime.now(pytz.UTC) - timedelta(hours=2)
+        self.user = UserFactory(username="test", email="test@example.com", password="123")
+
+    @patch(
+        'philu_commands.management.commands.auto_generate_certificates_for_open_courses._is_eligible_for_certificate',
+        return_value=False)
+    @patch('philu_commands.management.commands.auto_generate_certificates_for_open_courses.'
+           'is_course_valid_for_certificate_auto_generation', return_value=True)
+    @patch('philu_commands.management.commands.auto_generate_certificates_for_open_courses.modulestore')
+    @patch(
+        'philu_commands.management.commands.auto_generate_certificates_for_open_courses.CourseEnrollment.objects.filter'
+    )
+    @patch('philu_commands.management.commands.auto_generate_certificates_for_open_courses._get_cert_data')
+    @patch('philu_commands.management.commands.auto_generate_certificates_for_open_courses.generate_user_certificates',
+           return_value="generating")
+    @patch('philu_commands.management.commands.auto_generate_certificates_for_open_courses.log.info')
```

Comment:
Can we be consistent with the line breaks? Like if a patch spans on more than 1 line we can follow a consistency like maybe break right after the opening parentheses and then return value on next line and then ending parentheses on a new line?